test/GoogleAnalyticsEventProcessor/buildRequestUrl_test.js

```diff
@@ -0,0 +1,87 @@
+goog.module('measurementLibrary.eventProcessor.testing.GoogleAnalyticsEventProcessor.buildRequestUrl');
+goog.setTestOnly();
+
+const GoogleAnalyticsEventProcessor = goog.require('measurementLibrary.eventProcessor.GoogleAnalyticsEventProcessor');
+
+describe('The `buildRequestUrl` method ' +
+    'of GoogleAnalyticsEventProcessor', () => {
+  /**
+   * Tests the `buildRequestUrl` function by passing in an API secret,
+   * measurement ID, and measurement URL to construct the event processor
+   * with and comparing the result of the function with the expected result.
+   * @param {string} apiSecret
+   * @param {string} measurementId
+   * @param {string} measurementUrl
+   * @param {string} expected
+   */
+  function checkRequestUrl(apiSecret, measurementId, measurementUrl, expected) {
```

Comment:
This is totally optional, but I would probably name this "assertRequestUrl" to highlight that it's a test assertion and more importantly, that it throws.  Or if you really want to fit into the Jasmine syntax, change this to "expectRequestUrl", remove the "expected" param, and return the expectation object.

e.g. calls would look like:
```
expectRequestUrl('secret', 'measurementId', 'measurementUrl').toBe('http://...');
```